src/EditorFeatures/CSharpTest/Completion/CompletionProviders/XmlDocumentationCommentCompletionProviderTests.cs

```diff
@@ -246,16 +246,58 @@ public class foo
 }", "returns");
         }
 
+        [WorkItem(8627, "https://github.com/dotnet/roslyn/issues/8627")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ReadWritePropertyReturns()
+        {
+            await VerifyItemExistsAsync(@"
+public class foo
+{
+    
+    /// $$
+    public void bar { get; set; }
+}", "returns");
+        }
+
+        [WorkItem(8627, "https://github.com/dotnet/roslyn/issues/8627")]
+        [Fact, Trait(Traits.Feature, Traits.Features.Completion)]
+        public async Task ReadOnlyPropertyReturns()
+        {
+            await VerifyItemExistsAsync(@"
+public class foo
+{
+    
+    /// $$
+    public void bar { get; }
```

Comment:
```public void bar``` looks super weird :)   Can yiou change this test, unless you're actually verifying that 'void' properties get proper doc  comments.